migrations/schema.sql

```diff
@@ -298,6 +298,10 @@ BEGIN
         alter default privileges in schema graphql grant all on tables to postgres, anon, authenticated, service_role;
         alter default privileges in schema graphql grant all on functions to postgres, anon, authenticated, service_role;
         alter default privileges in schema graphql grant all on sequences to postgres, anon, authenticated, service_role;
+
+        -- Allow postgres role to allow granting usage on graphql and graphql_public schemas to custom roles
+        grant usage on schema graphql_public to postgres with grant option;
+        grant usage on schema graphql to postgres with grant option;
```

Comment:
comparing to `migrations/db/migrations/20220404205710_pg_graphql-on-by-default.sql` this looks like the substantive change, is that right?

Reply:
> Did you confirm that the postgres role was able to grant everything needed to execute a graphql request to a new role

I tested with psql:

1. `psql -U postgres`
2. `create role new_role;`
3. `grant new_role to postgres;`
4. `set role new_role;`
5. `select graphql.resolve($${ __schema { queryType { name } } } $$);` -- fails with 'permission denied for schema graphql'.
6. `reset role;`
7. `grant usage on schema graphql to new_role;`
8. `grant usage on schema graphql_public to new_role;`
9. `set role new_role;`
10. `select graphql.resolve($${ __schema { queryType { name } } } $$);` -- succeeds

> this looks like the substantive change, is that right?

Yes, correct.